browse/templates/base.html

```diff
@@ -6,7 +6,7 @@
   {%- block head -%}
   <title>{% block title %}{% endblock %}</title>
   <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
-  <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/css/arXiv-20190208.css?v=20190208" />
+  <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/css/arXiv-ARXIVNG-1918.css?v=20190307" />
```

Comment:
Is ``arXiv-ARXIVNG-1918.css`` right? Does it exist?

Reply:
Thanks for pointing this out. It does exist. Once I deploy browse and confirm all is OK, I'll update the "master" copy arXiv.css in the CDN and point browse to that. Want to minimize issues due to caching.